content/docs/0.8.1/references/settings.md

```diff
@@ -45,6 +45,8 @@ To configure Longhorn before installing it, see [this section](../../advanced-re
 #### Create Default Disk on Labeled Nodes
 > Example: `false`
 
+> Default: Un-checked
```

Comment:
Let's remove `Example` and replace it with `Default` if there is a default value to the setting. If not, we can add an `Example` after the `Default` value. And for boolean type, use `true` or `false` is better since it's not only observed from the UI.